### ISSUE TITLE
I'm unable to download the files

### ISSUE THREAD

**Issue by Techscrybe** [timsutton/brigadier#108]
When I clicked on the file to download, it just kept loading and the download didn't start.

I've tried it on two different devices but still thesame

**Comment by Juicyousiguy** [comment]
Hi, I was having this problem too.
Have you tried running it as an admin? That seemed to fix it for me.
If you have though, I wouldn't exactly know what to do. This program makes absolutely no sense to me.

**Comment by Techscrybe** [comment]
No, I don't need it anymore because the PC just broke down (not because of
the file tho).

It's an abandoned PC, so I was trying to revive it.
Thanks for your help, I really appreciate it.

P.S: have you tried downloading it from Apple official website? I've used
bootcamp in the past, and this is where I downloaded it.

**Comment by Juicyousiguy** [comment]
You can download it directly from the link brigadier shows you, and it will fully download, but you have to extract it yourself which, apparently depending on your 7-Zip version, will either work perfectly or literally not work in the slightest. I'm not sure about this myself though, it could just be that Apple is proudly serving up corrupted files.

**Comment by Techscrybe** [comment]
Thanks, I'll check it out.

**Comment by Technet8394** [comment]
What version of 7Zip is it?

**Comment by Juicyousiguy** [comment]
I'm not sure it will work, but apparently 22.01 (only avaliable on the GitHub page) works with brigadier and can extract the files properly.

**Comment by Technet8394** [comment]
worked!
